jmespath/jputil/util.go

```diff
@@ -0,0 +1,165 @@
+package jputil
```

Comment:
I'd suggest collapsing these fields down into the jmespath package. They don't seem to provide any usage outside of the jmespath project.  Them being exposed like this suggested they might be useful to the user of the library.